docs/modules/language-tutorial/pages/02_filling_out_a_template.adoc

```diff
@@ -32,8 +32,8 @@ parrot = (bird) {
 }
 ----
 
-Parrot and Pigeon have nearly identical properties.
-They only differ in their name and taxonomy, so if you have already written out `bird`, you can say that `parrot` is just like `pigeon` except `name` is `"Parrot"`, diet is `"Berries"` the `taxonomy.order` is `"Psittaciformes"`.
+`parrot` and `bird` have nearly identical properties.
```

Comment:
Same as above (although not critical); this refers to the named birds, not their describing properties.
```suggestion
Parrot and Pigeon have nearly identical properties.
```